src/gui_common/TweakedColourPicker.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Godot;
+
+/// <summary>
+///   Tweaked color picker defines some custom ColorPicker functions.
+/// </summary>
+public class TweakedColourPicker : ColorPicker
+{
+    private readonly CheckButton hsvCheckButton;
+
+    private readonly CheckButton rawCheckButton;
+
+    private readonly Button addPresetButton;
+
+    private bool hsvModeDisabled;
+
+    private bool rawModeDisabled;
+
+    public TweakedColourPicker()
+    {
+        var baseNode = GetChild(4).GetChild(4);
+        hsvCheckButton = baseNode.GetChild<CheckButton>(0);
+        hsvCheckButton.Connect("toggled", this, nameof(OnHSVButtonToggled));
+        rawCheckButton = baseNode.GetChild<CheckButton>(1);
+        rawCheckButton.Connect("toggled", this, nameof(OnRawButtonToggled));
+
+        addPresetButton = GetChild(7).GetChild<Button>(0);
+        addPresetButton.Connect("mouse_entered", this, nameof(OnMouseEnteredAddPresetButton));
+    }
+
+    [Export]
+    public bool HSVModeDisabled
```

Comment:
These two properties can then just be `ColorModeToggleEnabled` which determines the toggle buttons visibility.

Reply:
Well, HSV mode doesn't cause issues so can be reserved, while raw mode does cause problem so needs to be removed.